coralproject/talk
# Load more comments

## What does this PR do?

Loads only 10 comments initially and adds a "load more" button to load more in batches of 10.

## How do I test this PR?

1. Go to a comment stream with less than 10 comments.
2. Load More button should not be visible.
3. Comment until there are > 10 comments.
4. Load more button should be visible.
5. Clicking the button should load more comments.
